chat-widget/src/components/livechatwidget/common/reconnectChatHelper.ts

```diff
@@ -31,22 +32,26 @@ const handleChatReconnect = async (chatSDK: any, props: any, dispatch: Dispatch<
     //Redirect if enabled
     if (reconnectChatContext?.redirectURL) {
         redirectPage(reconnectChatContext.redirectURL, props.reconnectChatPaneProps?.redirectInSameWindow);
-        return;
+        return false;
     }
 
     if (hasReconnectId(reconnectChatContext)) {
         //if reconnect id is provided in props, don't show reconnect pane
         if (props.reconnectChatPaneProps?.reconnectId && !isNullOrEmptyString(props.reconnectChatPaneProps?.reconnectId)) {
-            await setReconnectIdAndStartChat(isAuthenticatedChat, chatSDK, props, dispatch, setAdapter, reconnectChatContext.reconnectId ?? "", initStartChat);
-            return;
+            await setReconnectIdAndStartChat(isAuthenticatedChat, chatSDK, state, props, dispatch, setAdapter, reconnectChatContext.reconnectId ?? "", initStartChat);
```

Comment:
Why are we introducing `state` as a parameter? Is it because we are including the scenario to set custom context?

Reply:
"state" is required in initStartChat. It is a bug for omitting it here